i18n/translations/en/form-builder.json

```diff
@@ -619,11 +619,34 @@
   "settings": {
     "formManagement": "Form management",
     "api": {
+      "keyId": "API Key ID",
       "title": "API Access",
-      "generateKey": "Generate Key",
+      "generateKey": "Generate API Key",
       "refreshKey": "Refresh Key",
-      "deleteKey": "Delete Key",
-      "keyExists": "API Key Exists"
+      "deleteKey": "Delete API Key",
+      "keyExists": "API Key Exists",
+      "keyIdToolTip": {
+        "text1": "What is an API key ID?",
+        "text2": "The API key ID is not the API key itself. It provides a safe way to identify a key without sharing protected information."
+      },
+      "dialog": {
+        "title": "API key successfully created",
+        "heading": "GC Forms API keys are PROTECTED B.",
+        "responsibility": "It is your responsibility to:",
+        "responsibility1": "Save this key in a secure location.",
+        "responsibility2": "Share the key only with people who need it.",
+        "responsibility3": "Send the key through secure channels, like encrypted email.",
+        "responsibility4": "Read and the understand the <a href='canada.ca' target='_blank'>required security procedures.</a>",
+        "text1": "Do not loose your key",
+        "text2": "This is your unique password and it is non-recoverable. If you loose this key, you will have to create a new one.",
+        "cancelButton": "Cancel",
+        "downloadButton": "Download key",
+        "confirm": {
+          "label": "Confirmation",
+          "required": "(required)",
+          "description": "Type <strong>AGREE</strong> in the box below to confirm you understand your responsibilities and agree to the <a href='canada.ca' target='_blank'>required security procedures</a> while storing and sharing this key."
+        }
+      }
```

Comment:
```suggestion
      "deleteKey": "Delete API key",
      "keyExists": "API key exists",
      "keyIdToolTip": {
        "text1": "What is an API key ID?",
        "text2": "The API key ID is not the API key itself. It provides a safe way to identify a key without sharing protected information."
      },
      "dialog": {
        "title": "API key successfully created",
        "heading": "GC Forms API keys are PROTECTED B",
        "responsibility": "It is your responsibility to:",
        "responsibility1": "Save this key in a secure location.",
        "responsibility2": "Share the key only with people who need it.",
        "responsibility3": "Send the key through secure channels, like encrypted email.",
        "responsibility4": "Understand the <a href='canada.ca' target='_blank'>required security procedures.</a>",
        "text1": "Do not lose your key",
        "text2": "This key is provided once as it can give access to form response data. If you lose this key, or if it becomes compromised, you will have to create a new one.",
        "cancelButton": "Cancel",
        "downloadButton": "Download API key",
        "confirm": {
          "label": "Confirmation",
          "required": "(required)",
          "description": "Type <strong>AGREE</strong> in the box below to confirm you understand your responsibilities and agree to the <a href='canada.ca' target='_blank'>required security procedures</a> while storing and sharing this key."
        }
      }
```